great_expectations/execution_engine/sqlalchemy_execution_engine.py

```diff
@@ -249,7 +250,38 @@ def _get_dialect_type_module(dialect):
     return dialect
 
 
+@public_api
 class SqlAlchemyExecutionEngine(ExecutionEngine):
+    """SparkDFExecutionEngine instantiates the ExecutionEngine API to support computations using Spark platform.
+
+    Constructor builds a SqlAlchemyExecutionEngine, using a provided connection string/url/engine/credentials to \
+    access the desired database.
+
+    Also initializes the dialect to be used.
+
+        Args:
+            name (str): The name of the SqlAlchemyExecutionEngine
+            credentials: If the Execution Engine is not provided, the credentials can be used to build the \
+                ExecutionEngine object. If the Engine is provided, it will be used instead.
+            data_context (DataContext): An object representing a Great Expectations project that can be used to \
+                access ExpectationSuite objects and the Project Data itself.
+            engine (Engine): A SqlAlchemy Engine used to set the SqlAlchemyExecutionEngine being configured, \
+                useful if an Engine has already been configured and should be reused. Will override Credentials if \
+                provided.
+            connection_string (string): If neither the engines nor the credentials have been provided, a \
+                connection string can be used to access the data. This will be overridden by both the engine and \
+                credentials if those are provided.
+            url (string): If neither the engines, the credentials, nor the connection_string have been provided, a \
+                URL can be used to access the data. This will be overridden by all other configuration options if \
+                any are provided.
+            concurrency (ConcurrencyConfig): Concurrency config used to configure the sqlalchemy engine.
```

Comment:
```suggestion
    Args:
        name (str): The name of the SqlAlchemyExecutionEngine
        credentials: If the Execution Engine is not provided, the credentials can be used to build the \
            ExecutionEngine object. If the Engine is provided, it will be used instead.
        data_context (DataContext): An object representing a Great Expectations project that can be used to \
            access ExpectationSuite objects and the Project Data itself.
        engine (Engine): A SqlAlchemy Engine used to set the SqlAlchemyExecutionEngine being configured, \
            useful if an Engine has already been configured and should be reused. Will override Credentials if \
            provided.
        connection_string (string): If neither the engines nor the credentials have been provided, a \
            connection string can be used to access the data. This will be overridden by both the engine and \
            credentials if those are provided.
        url (string): If neither the engines, the credentials, nor the connection_string have been provided, a \
            URL can be used to access the data. This will be overridden by all other configuration options if \
            any are provided.
        concurrency (ConcurrencyConfig): Concurrency config used to configure the sqlalchemy engine.
```